theforeman/forklift
# Adding a way to copy custom files to a dev box

This allows devs to specify custom files (like dotfiles) to be copied over to `~/vagrant` when spinning up or provisioning a development environment. These files can be added to `roles/customize_home/files` and they will be copied over. If that folder does not exist, the copy step is skipped, so a developer can choose to ignore this feature too.

To test:

1. create `roles/customize_home/files`
2. Add files there
3. Spin up a `centos7-katello-devel` or `centos7-katello-devel-stable` environment after copying over the box definition
4. Make sure the files are copied over to `~/vagrant` on the devel environment
